### PR TITLE
Adding smaller test data

### PR DESCRIPTION
**Pull request by BFedder**
I'm for now just using this PR to trigger codecov, as this is the fastest way I know to determine coverage.

To make the pyedr test files smaller, I added the edr test file I created for the EDRReader and changed the tests for pyedr accordingly. The tests now pass, but I want to make sure that the much smaller file still triggers all of the same code to be tested.

### CODE REVIEW

**Comment by BFedder** [comment]
Okay, so coverage of panedr.py and pyedr.py appears to be unaffected. Should we replace the test data files then, or am I missing a reason why that would be a bad idea?

**Comment by hmacdope** [comment]
Not in my mind as long as you clarify how the files are different (both here and possible in a README also?)

**Comment by IAlibay** [comment]
@hmacdope - It all depends on how small the package gets with it, if the package is still > a couple of MB when inflated then it's not worth it. The whole point of pyedr was that pandas was too heavy a dependency. We're breaking that idea if we just end up adding more tests.

If we don't package the tests then reducing the test sizes don't matter (in fact you just end up using more git history space).

**Comment by BFedder** [comment]
The new files are from a very short simulation of lysozyme that I ran just to get them. If we do go ahead and replace cat.edr and cat.xvg with the new files, then this would reduce the test data size from 29 MB to 0.5 MB. 

As far as I can tell, there is no practical difference, as the coverage of panedr.py and pyedr.py is unchanged. The new files do only contain 4 frames (down from 14100), however, so I am not sure how true of a test of output verbosity this still poses.

**Comment by IAlibay** [comment]
>  then this would reduce the test data size from 29 MB to 0.5 MB.

That's reasonably small tbh, @jbarnoud would be the right judge of whether or not this is a suitable replacement for this test.

**Comment by jbarnoud** [comment]
I'm on my phone so it's a bit difficult to top better than work from memory. However, I think I remember that the point of the test file is to make sure we concatenate edr files the same way as gromacs. I don't see in the changes how it would have tested that, though 🤔

What I see, however is that the progress bar does not get tested as much as it was. Because it adapts to the number of frames, there should be a replacement test somehow to cover the feature.

**Comment by orbeckst** [comment]
Coverage also analyzed the test files. They should be excluded from coverage reporting.

**Comment by BFedder** [comment]
I have tried changing pyedr now so that the progress bar is provided by tqdm instead. This also allowed me to remove the tests for the progress bar. However, I can't get codecov to work properly now... Please could someone help?

**Comment by IAlibay** [comment]
> I can't get codecov to work properly now... Please could someone help?

Is this the upload failure or is there something else going wrong here?

File: panedr/requirements.txt
Comment (IAlibay) [inline panedr/requirements.txt:4]: Just FYI here - @BFedder and I had a chat about this a week or so ago and I told him to go ahead with this, tqdm is a reasonably small dependency, and it will make life a lot easier imho.

@jbarnoud as the original code author, do you have any issues with this?

**Review by IAlibay — CHANGES_REQUESTED** [review]
Couple of small things - could use a second opinion on removing timing @orbeckst @jbarnoud @hmacdope

File: pyedr/setup.cfg
Comment (IAlibay) [inline pyedr/setup.cfg:31]: Forgot to also say, can you add tqdm to the gh actions CI yaml file?

**Review by IAlibay — CHANGES_REQUESTED** [review]

File: pyedr/pyedr/pyedr.py
Comment (hmacdope) [reply pyedr/pyedr/pyedr.py:467]: I think there isn't really a need for timing as a default or even verbose option. IMO TQDM has enough timing info.

Comment (IAlibay) [reply pyedr/pyedr/pyedr.py:467]: I guess you'll still need the verbose flag to turn tqdm on/off?

File: pyedr/pyedr/pyedr.py
Comment (BFedder) [reply pyedr/pyedr/pyedr.py:460]: Beautiful, thank you!

**Review by IAlibay — APPROVED** [review]
lgtm - assuming @jbarnoud is fine with the changes here

**Review by IAlibay — CHANGES_REQUESTED** [review]
O actually forgot - @BFedder can you add an entry to the changelog? (you'll need to add a new version entry)

**Comment by BFedder** [comment]
Done! 
Also deleted the old test data files now.

**Review by IAlibay — APPROVED** [review]
thanks @BFedder

**Review by hmacdope — APPROVED** [review]

**Pull request merged**
